File: problems/problemset/ttl-cache/Python/template/ttl_cache.py
class TTLCache:
    def __init__(self):
        pass

    def set(self, key: str, value: str, ttl: int) -> None:
        pass

    def get(self, key: str) -> str:
        pass

    def delete(self, key: str) -> None:
        pass

    def size(self) -> int:
        pass

    def tick(self, delta: int) -> None:
        pass

File: problems/problemset/ttl-cache/Rust/template/ttl_cache.rs
pub struct TTLCache {
    // Add your fields here
}

impl TTLCache {
    pub fn new() -> Self {
        todo!()
    }

    pub fn set(&mut self, key: &str, value: &str, ttl: i64) {
        todo!()
    }

    pub fn get(&self, key: &str) -> String {
        todo!()
    }

    pub fn del(&mut self, key: &str) {
        todo!()
    }

    pub fn size(&self) -> usize {
        todo!()
    }

    pub fn tick(&mut self, delta: i64) {
        todo!()
    }
}
